scripts/integrity.js

```diff
@@ -75,4 +77,4 @@ for (var i = 0; i < config.fontawesome.length; i++) {
     }
 }
 
-fs.writeFileSync(configFile, yaml.dump(config));
+fs.writeFileSync(configFile, yaml.dump(config, { lineWidth: 110 }));
```

Comment:
Just a JS yaml option which is why I ask you if we should stick with the
default because that is the cause for the changes in config.

On Aug 5, 2016 00:49, "Joshua Mervine" <notifications@github.com> wrote:

> In scripts/integrity.js
> <https://github.com/MaxCDN/bootstrap-cdn/pull/727#discussion_r73609216>:
>
> > @@ -75,4 +77,4 @@ for (var i = 0; i < config.fontawesome.length; i++) {
> >      }
> >  }
> >
> > -fs.writeFileSync(configFile, yaml.dump(config));
> > +fs.writeFileSync(configFile, yaml.dump(config, { lineWidth: 110 }));
>
> I read over everything else and it looks good. I just wasn't familiar with
> the lineWidth option. I'll google. :P
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MaxCDN/bootstrap-cdn/pull/727/files/bc06fe1d74c5cbf81dc4ec2cda7c998c2c684dd3#r73609216>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtQt6NDtA47ZQ6P2cLX0kpDrrY7Jlks5qcl5dgaJpZM4JcmoI>
> .
>